test/integration/targets/ec2_eip/tasks/main.yml

```diff
@@ -8,7 +8,7 @@
           aws_secret_key: "{{ aws_secret_key }}"
           security_token: "{{ security_token }}"
           region: "{{ aws_region }}"
-      no_log: True
+      no_log: False
```

Comment:
Please don't do this: we really don't want these values logged into shippable.

Personally I'm a big fan of using `defaults: { group/aws: { .... } }` rather than the set_fact block since modules know what they shouldn't log

Reply:
Oversight from my tests, thanks.

Reply:
> Personally I'm a big fan of using `module_defaults: { group/aws: { .... } }` rather than the set_fact block since modules know what they shouldn't log

Plus an efficient way to factor the credential lines :+1: